src/server/telemetry.js

```diff
@@ -13,16 +12,30 @@ var PLUGIN_PROPS_TO_ANONYMIZE = [
 ];
 
 /**
- * Handles telemetry sent by clients. Anonymizes properties if necessary, and sends the event to the external telemetr module if one was provided.
- * 
+ * @param {object=} opts
+ * @constructor
+ */
+function Telemetry(simulator, opts) {
+    this._simulator = simulator;
```

Comment:
This change is pending, I'm improving how telemetry module is created and initialized so I can pass only the project. 
At the point in which this object is created, the project instance is not ready yet, and in order to create a project instance, we need the telemetry instance ready.